src/modules/SX128x/SX128x.cpp

```diff
@@ -996,8 +996,14 @@ int16_t SX128x::setSyncWord(const uint8_t* syncWord, uint8_t len) {
     this->syncWordLen = len;
   }
 
+  uint16_t syncAddr = len == 5 ? RADIOLIB_SX128X_REG_SYNC_WORD_1_BYTE_4 :
```

Comment:
This is bit too complex (and will make possible future extension into more sync words harder). What this really does is `RADIOLIB_SX128X_REG_SYNC_WORD_1_BYTE_4 + (5 - len)`.

Reply:
> possible future extension into more sync

friendly reminder that 5 bytes is the maximum limit that SX1280  can do

idea was to make the code independent of actual RADIOLIB_SX128X_REG_SYNC_WORD_1_BYTE_x addresses

I prefer to make the source code readable by humans and to let the modern compilers do the binary code optimization job.

Reply:
updated with one more commit

Reply:
Great, thank you! Usually I try not to leave too much up to the magic black optimization box - and I'm really not sure those ternary operators would optimize to addition.